src/Sentry.Unity/ScriptableSentryUnityOptions.cs

```diff
@@ -120,12 +121,12 @@ public static string GetConfigPath(string? notDefaultConfigName = null)
     /// <remarks>
     /// Used for loading the SentryUnityOptions from the ScriptableSentryUnityOptions during runtime.
     /// </remarks>
-    public static SentryUnityOptions? LoadSentryUnityOptions(ISentryUnityInfo unityInfo)
+    public static SentryUnityOptions? LoadSentryUnityOptions()
```

Comment:
We no longer need to provide the `IUnityInfo` since these are getting set up during startup as part of the new platform services that are statically available.